permits/migrations/0032_add_notification_for_services_in_wot.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.4 on 2021-07-13 18:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('permits', '0031_add_mandatory_2fa_to_permitdepartment'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='worksobjecttype',
+            name='notify_services',
+            field=models.BooleanField(default=False, verbose_name='Notifier les services'),
+        ),
+        migrations.AddField(
+            model_name='worksobjecttype',
+            name='services_to_notify',
+            field=models.TextField(blank=True, verbose_name='Emails des services à notifier'),
```

Comment:
It seems that if you use space or line return, the emails won't be separated correctly in the permits view.
So I would add a helptext : 
Veuillez séparer les emails par une virgule ","

Reply:
Done!